tests/components/tasmota/conftest.py

```diff
@@ -0,0 +1,43 @@
+"""Test fixtures for Tasmota component."""
+
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.tasmota.const import (
+    CONF_DISCOVERY_PREFIX,
+    DEFAULT_PREFIX,
+    DOMAIN,
+)
+
+from tests.common import MockConfigEntry, mock_device_registry, mock_registry
+
+
+@pytest.fixture
+def device_reg(hass):
+    """Return an empty, loaded, registry."""
+    return mock_device_registry(hass)
+
+
+@pytest.fixture
+def entity_reg(hass):
+    """Return an empty, loaded, registry."""
+    return mock_registry(hass)
+
+
+async def setup_tasmota(hass, entry=None):
```

Comment:
If we don't need to pass a specific entry, this function can be a pytest fixture.

Reply:
Fixed.